conformance/fhir-ig-davinci-pdex-formulary/src/test/resources/examples/200/Basic-FormularyItem-D1002-1049640.json

```
@@ -0,0 +1,129 @@
+{
+    "resourceType": "Basic",
+    "id": "FormularyItem-D1002-1049640",
+    "meta": {
+        "lastUpdated": "2021-08-22T18:36:03.000+00:00",
+        "profile": [
+            "http://hl7.org/fhir/us/davinci-drug-formulary/StructureDefinition/usdf-FormularyItem"
```

lets go ahead and add the version suffix to all the 2.0 examples:
```suggestion
            "http://hl7.org/fhir/us/davinci-drug-formulary/StructureDefinition/usdf-FormularyItem|2.0.0"
```

---

This has been fixed by adding version suffix to all 2.0 examples.